Cargo.toml

```diff
@@ -12,10 +12,15 @@ readme = "README.md"
 toml = "0.2.0"
 quick-error = "1.1.0"
 clap = "2.10.4"
+cargo = "0.12.0"
+term = "0.4.4"
 
-[package.metadata.sphinx]   
+[package.metadata.sphinx]
 docs-path = "docs"
 commit-message = "(cargo-sphinx) Generate docs."
 sign-commit = false
 push-remote = "origin"
 push-branch = "gh-pages"
+
+[profile.dev]
+lto = true
```

Comment:
I may roll this back. My understanding from The Language That Shall Not Be Names is that this that Link Time Optimization is a technique to move various optimizations that normally happen at compilation time to linking time. This allows for more efficient reuse of intermediate build artifacts that haven't actually been touched since the last compile. I haven't even `time`d the results here, but it doesn't seem to do much. That said, I've seen this in a bunch of different projects too, so it shouldn't do any harm.